ServerApp/app/user/controllers.py

```diff
@@ -0,0 +1,82 @@
+from sqlalchemy.exc import IntegrityError
+from app import *
+from .models import User
+
+mod_user = Blueprint('user', __name__)
+
+NET_SEC_PASSWD = 'PASSWD'
+NETWORK_SECRET = NET_SEC_PASSWD
+
+try:
+    logged_in = pickle.load(open('logged_in', 'rb'))
+except:
+    logged_in = []
+
+def is_logged_in(username):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [function already defined line 3](https://app.codacy.com/manual/rejahrehim/SecureTea-Project/pullRequest?prid=5573783)

Reply:
Didn't find re-definition in the file